CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 ## Unreleased
 
-* [FEATURE] Add `-kubelet-endpointslice` argument to support `EndpointSlice` for the kubelet controller.
+## 0.77.0 / 2024-09-18
+
+> Along with this release we also introduce a new command line tool [poctl](https://github.com/prometheus-operator/poctl). It is designed specifically for managing Prometheus Operator Custom Resources.
+
+> This project is currently under active development and fully experimental, so expect breaking changes and rough edges. We encourage you to [try it out](https://github.com/prometheus-operator/poctl/releases/tag/v0.1.0) and provide the feedback.
+
+* [CHANGE] Add more API validation and changes in ScrapeConfig CRD. #6921 #6855
+* [CHANGE] Update API validations for ECSDConfig in ScrapeConfig CRD. #6744
+* [FEATURE] Add experimental Prometheus 3 support. #6940
```

Comment:
I was wondering if also makes sense highlight Prometheus 3 is experimental and we're not supporting it, wdyt?